hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -1014,6 +1014,11 @@ static FSNamesystem loadFromDisk(Configuration conf) throws IOException {
       this.leaseRecheckIntervalMs = conf.getLong(
           DFS_NAMENODE_LEASE_RECHECK_INTERVAL_MS_KEY,
           DFS_NAMENODE_LEASE_RECHECK_INTERVAL_MS_DEFAULT);
+      Preconditions.checkArgument(
+              leaseRecheckIntervalMs >= 0,
```

Comment:
Should this be `> 0` ? Is it valid to have a sleep time of zero in this case, which would make it loop continuously?

Reply:
`> 0` is better, I will modify it.